native/src/redux/reducers/__tests__/cityContentReducer.spec.ts

```diff
@@ -17,6 +17,8 @@ import { CATEGORIES_ROUTE, CategoriesMapModel, ErrorCode, EVENTS_ROUTE, Language
 import cityContentReducer from '../cityContentReducer'
 import { CityContentStateType } from '../../StateType'
 
+jest.mock('rn-fetch-blob')
+
```

Comment:
Why did you have to/why did you add those mocks?

Reply:
I looked a little bit more into this. You need to mock it here as we are importing any dependency from `helpers.ts` in this module or any module imported in this module. In the `helpers.ts` at the beginning of the file we are importing `RNFetchBlob from 'rn-fetch-blob`. Withouth a mock this module will throw.
But now if we take a look at the `jest.setup.ts` we see that we actually do mock `rn-fetch-blob` there. 
However, I can tell this mock there will do nothing really. What happens in the `setup.jest.ts` is that we let jest generate a mock for `rn-fetch-blob`. But I think what we want is that jest uses our mock for the test files. To enable this behavior we can remove `jest.mock('rn-fetch-blob')` from the `jest.setup.ts` and add `jest.doMock('rn-fetch-blob')` below the call to walkDir, as we first have to unmock the automatic mock that jest creates for all our files. It is also important to call `doMock` instead of just `mock` as the `jest.mock` statements are all hoisted to the beginning of the file, meaning before we unmocked our `rn-fetch-blob` module.

After doing that you can basically remove all jest.mock('rn-fetch-blob') from the other files, as we then have a global mock for this module.

Reply:
Thanks Sarah for checking this out! Makes it a lot nicer :)